core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -67,10 +67,13 @@ static PageStore create(PageStoreOptions options) {
    *
    * @param pageId page identifier
    * @param page page data
+   * @param isTemporary is page data temporary
    * @throws ResourceExhaustedException when there is not enough space found on disk
    * @throws IOException when the store fails to write this page
    */
-  void put(PageId pageId, byte[] page) throws ResourceExhaustedException, IOException;
+  void put(PageId pageId,
+      byte[] page,
+      boolean isTemporary) throws ResourceExhaustedException, IOException;
```

Comment:
`putTemporary` instead of adding a boolean parameter?

Reply:
done